src/plugins/auto/auto_schedule.cpp

```diff
@@ -234,20 +234,14 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
     bool isCumulative =
         (_autoSContext->_performanceHint == IE::PluginConfigParams::CUMULATIVE_THROUGHPUT) ? true : false;
     if (isCumulative) {
-        std::list<DeviceInformation> validDevices =
-            _autoSContext->_plugin->GetValidDevice(_autoSContext->_devicePriorities,
-                                                   _loadContext[ACTUALDEVICE].networkPrecision);
```

Comment:
why not call getvaliddevice

Reply:
will get empty list after calling this function if the inputting priority list is "mock.o,mock.1,......".

Reply:
why,auto is using black list, is mock in black list?

Reply:
No. It's not caused by black list. From my understand, function GetVaildDevice() just parses out the devices based on selection priority like  "dGPU > VPUX > iGPU > MYRIAD > CPU", which didn't include the mock device.